src/ngraph/op/result.cpp

```diff
@@ -28,15 +28,15 @@ op::Result::Result(const shared_ptr<Node>& arg)
     : Op("Result", check_single_output_args({arg}))
 {
     constructor_validate_and_infer_types();
+    // always borrow the placement conf even the default one
+    set_placement_index(get_argument(0)->get_placement_index());
```

Comment:
@rkimballn1 do we need this? DefaultPlacement pass seems to be doing the backend assignment anyway.

Reply:
This should be replaced by a pass where/if it's needed, but can be a later PR.

Reply:
Leave that there for now